### ISSUE TITLE
npm publish doesn't change npm create templates

### ISSUE THREAD

**Issue by akabekobeko** [uetchy/create-create-app#27]
I updated the `templates/` with [create-book](https://github.com/vivliostyle/create-book) using `create-create-app` and then `npm pulish`, but the one generated by running `npm create book` is still the old templates.

Is there anything else need to do to update the image run by `npm create` other than `npm publish`?

`README.md` explains that `npm publish` is running.

> ### 3. Publish package to npm
>
> Run `npm publish` or `yarn publish` to publish your `create-greet` app to npm.

When I installed the published one with `npm i create-book`, npm was up to date.

**Comment by uetchy** [comment]
quick search resulted in this issue:
https://github.com/npm/cli/issues/2395

looks like that behavior has been fixed in v7.8.0, which was published just 5 days ago.
https://github.com/npm/cli/commit/37613e4e686e4891210acaabc9c23f41456eda3f

Try updating npm and see the problem persists

**Comment by akabekobeko** [comment]
Thank you. When I checked the version of npm in the environment where `npm publish` was done, it was `7.7.6`.

Let's update npm and then check.

**Comment by akabekobeko** [comment]
I updated the version of create-book to `npm publish` in an environment where npm was updated to `7.8.0`.

```
$ npm -v
7.8.0
```

And when I ran `npm create book sample` in the same environment, the old `templates/default` was installed. After that, when I ran the following, the published create-book was up to date.

```
$ cd sample
$ npm i create-book
```

So it doesn't seem to be fixed just by updating npm `7.8.0`. Since it is a problem of npm cli, we will observe it at a fixed point.
